GVRf/Sample/gvr-video/src/org/gearvrf/video/VideoScript.java

```diff
@@ -410,6 +410,12 @@ public void run() {
                     mGVRContext, "button/play-active.png"));
             mPlayPauseButton = new GVRSceneObject(gvrContext,
                     gvrContext.createQuad(0.7f, 0.7f), mInactivePause);
+            mPlayPauseButton.getRenderData().getMaterial().setTexture("active_play", mActivePlay);
```

Comment:
No, I don't think so. It's `setKeepWrapper()` not `setKeepAlive()` precisely because we did already have a mechanism for artificially inflating reference counts ... and because `setKeepWrapper()` _is_ rather hacky and not something we should be encouraging.